FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_DNS.c

```diff
@@ -1333,7 +1333,7 @@ const size_t uxAddressLength = ipSIZE_OF_IPv4_ADDRESS;
 							 */
 							pvCopySource = &pucByte[ sizeof( DNSAnswerRecord_t ) ];
 							pvCopyDest = &ulIPAddress;
-							( void ) memcpy( pvCopyDest, pvCopySource, uxAddressLength );
+							( void ) memcpy( pvCopyDest, pvCopySource, sizeof( uint32_t ) );
```

Comment:
Oops, I did intend to use `uxAddressLength` in stead of `sizeof()`.
This has to do with the IPv6 release, in which `uxAddressLength` is variable ( 4 or 16 ).

Reply:
I see. But the `ulIPAddress` is defined as a 32 bit variable. And if we copy more than 4 bytes, do we not risk overwriting some other memory?
Is this intended? Or shall we be using some different datatype for `ulIPAddress`?

Reply:
Why would we copy more than 4 bytes?
There is:
~~~c
    #define ipSIZE_OF_IPv4_ADDRESS     4U  /* See FreeRTOS_IP.h */
    const size_t uxAddressLength = ipSIZE_OF_IPv4_ADDRESS;
~~~
and so the `memcpy()` will copy 4 bytes only.

> Or shall we be using some different datatype for ulIPAddress
No, that is an ordinary IPv4 address, 4 bytes long.

Reply:
I see. I was talking about this line from your comment "_This has to do with the IPv6 release, in which uxAddressLength is variable ( 4 or 16 )._"
What is 4 or 16? I assume that is the value in uxAddressLength. If it is 4, then it is fine, if it is 16, then we would be in trouble...

Reply:
As this is not interesting for the PR, I will reply by email.
Hein